coala_quickstart/coala_quickstart.py

```diff
@@ -7,6 +7,7 @@
 
 from coalib.output.printers.LogPrinter import LogPrinter
 
+from coala_utils.FilePathCompleter import FilePathCompleter
```

Comment:
Yep, the comment https://github.com/coala/coala-quickstart/pull/172#issue-249149958 mentions it :)